src/pyfaf/actions/fedmsg_notify.py

```diff
@@ -65,6 +65,7 @@ def run(self, cmdline, db):
                                   q_yesterday.c.sum_yesterday)
                  .outerjoin(q_today, Report.id == q_today.c.t_report_id)
                  .outerjoin(q_yesterday, Report.id == q_yesterday.c.y_report_id)
+                 .filter(and_(Report.max_certainty.isnot(None), Report.max_certainty != 100))
```

Comment:
Correct! Reports that does not have a solution should be notified through fedmsg, right?